Makefile

```diff
@@ -164,14 +180,19 @@ MACHINE_SIMPLE-$1=$$(MACHINE_SIMPLE-$$(ARCH-$1))
 else
 MACHINE_SIMPLE-$1=$$(ARCH-$1)
 endif
-SDK-$1=$$(basename $1)
+
+MACOSX_DEPLOYMENT_TARGET=10.6
```

Comment:
@dimaqq You need to install old SDK's manually in any case. See  [xcodelegacy](/devernay/xcodelegacy).